testsuite/tests/project_support/test.sh

```diff
@@ -0,0 +1,8 @@
+# Reduce in directory dir1
+(cd dir1 ; $ADAREDUCER p.gpr oracle.sh > /dev/null)
+
+# Check that dir1/hello.adb has been minimized
+grep "with P" dir1/hello.adb
```

Comment:
Do we have a proper heuristic to know whether a file was reduced?

Reply:
I don't think it's easy to come up with one for complex cases - the criteria for proper minimization will be a moving target as adareducer becomes better at its job. For this particular test it doesn't matter much, this is a sanity check to verify that _something_ was done on hello.adb, the important test is to verify that something was done in the other directory, which we couldn't have found without proper project support.